src/api/users/repository.ts

```diff
@@ -80,6 +80,50 @@ export class UsersRepository implements Repository<User> {
     return Promise.reject('Not implemented yet');
   }
 
+  async updateGalleryUsage(userId: UserId, sizeIncrement: GalleryUsage) {
```

Comment:
The repository should not handle any use-case logic. This should be in the use case, the repository just interacts with the database.